Data/ByteString/Internal.hs

```diff
@@ -235,11 +235,12 @@ instance Ord ByteString where
 
 instance Semigroup ByteString where
     (<>)    = append
+    {-# INLINABLE sconcat #-}
```

Comment:
What's the motivation for adding this pragma in the context of this PR?

Reply:
It's not very well-motivated. But it is the only instance method definition that is part of the now-untangled mutually recursive group and produces an un-elided binding that can be marked `INLINABLE`; and I'm still not clear on the conditions under which such markings are desirable.